services/spark-operator/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+displayName: Spark
+description: Apache Spark is a fast and general-purpose cluster computing system for big data
+category:
+  - compute
+  - bigdata
+overview: >
+  # Overview
+  Apache Spark is a fast and general-purpose cluster computing system for big data. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including: Spark SQL for SQL and DataFrames, MLlib for machine learning, GraphX for graph processing, and Spark Streaming for stream processing.
+
+  - [Spark Documentation](https://spark.apache.org/docs/latest/)
+icon: 
+https://raw.githubusercontent.com/mesosphere/kubeaddons-enterprise/dev/metadata/static/spark/logo.svg
```

Comment:
Not sure which ican url we should use? And also I guess we are supposed to base64 this?

Reply:
Yes to base64 - maybe we can also loop in UX to okay the actual icon. @TattdCodeMonkey any tips here?

Reply:
Got the 🆗  from Nan on the graphic content of the svg above. Now I put the b64 encoded version of the text content of the link above.